Skype/SDN/articles/tenantid-element-byeendpointtype-complextype-skype-for-business-sdn-interface-2-2-schema-d.md

```diff
@@ -26,8 +27,8 @@ Elements and attributes
 
 <table>
 <colgroup>
-<col style="width: 50%" />
-<col style="width: 50%" />
+<col/>
+<col/>
```

Comment:
No need to change anything.  Just a note.  For this type of issue, you can also delete rows 29-32, which gets rid of the HTML formatting for columns.  What you did is also resolves the issue.